bricksrc/equipment.py

```diff
@@ -464,10 +465,10 @@
 }
 
 security_subclasses = {
-    "Access_Control": {
+    "Access_Control_Equipment": {
         "tags": [TAG.Equipment, TAG.Security, TAG.Access, TAG.Control],
         "subclasses": {
-            "Access_Reader": {
+            "Access_Reader_Equipment": {
```

Comment:
Sorry for my ignorance in the previous comment. Could you take out `_Equipment` postfix here? I was meaning `Equipment` as a Tag but it's already there.

Reply:
Done